playwright/test/contexts.ts

```diff
@@ -1,8 +1,11 @@
 import { baseUrl, useStageApis } from './helpers/urls';
 import { Response } from 'playwright';
 
-export function gotoWithoutCache(url: string): Promise<null | Response> {
-  return page.goto(`${url}?cachebust=${Date.now()}`);
+export function gotoWithoutCache(
```

Comment:
What is this used for?

Reply:
This is used for all the Playwright tests, to make sure we're testing the most recent code. However, this change shouldn't have made it into this PR because I didn't do the testing this way in the end.

Reply:
To make sure e2e tests that happen immediately after deploy are hitting the new pages